.github/workflows/pr-custom-review.yml

```diff
@@ -0,0 +1,26 @@
+name: Check reviews
+
+on:
+  pull_request:
+    branches:
+      - master
+      - main
+    types:
+      - opened
+      - reopened
+      - synchronize
+      - review_requested
+      - review_request_removed
+  pull_request_review:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: pr-custom-review
+        uses: paritytech/pr-custom-review@v1
+        with:
+          token: ${{ secrets.PRCR_TOKEN }}
```

Comment:
according to what I've talked with @TriplEight on DMs, this token will not currently work because it was generated by a GitHub App and therefore it will expire. e.g. for installation tokens it expires in 1 hour

https://docs.github.com/en/developers/apps/building-github-apps/authenticating-with-github-apps#authenticating-as-an-installation

> Installation access tokens have the permissions configured by the GitHub App and expire after one hour.

Reply:
ASAIK it was PAT(personal access token). @TriplEight can you confirm whether its true or not?